### ISSUE TITLE
disabling react/jsx-no-literals and no-console does not work in files

### ISSUE THREAD

**Issue by ambewas** [babel/babel-eslint#519]
**Expected behaviour**
Adding /* eslint-disable react/jsx-no-literals */ to the top of the file should disable the rule for the file in question.

**Actual behavior**
The rule still gets triggered when running gulp-eslint with babel-eslint as a parser

**More information**
Other rules (eslint default rules, or from this plugin) can be disabled just fine. Only trying to disable this specific rule does not work.

This behaviour started today -- last friday everything was still fine. Gulp-eslint has not been updated since.

We're using babel-eslint as parser. I've tried pinning that one down to a previous version, but that didn't work. 

I'm having trouble understanding why this is occurring. It seems to be very random. 

Some things I've tried:

```
/* eslint-disable react/jsx-no-literals */

<Component>literal string</Component>
```
-> rule still triggers

So I then tried a global disable on top of the file:
```
/* eslint-disable */
<Component>literal string</Component>
```
-> rule still triggers

Next I noticed there's another rule that still gets triggered, leading me to believe this has to do with the parser:

```
/* eslint-disable no-console */

console.log('hello'); // -> does not trigger the rule

<Component
  onClick={console.log('hello')} // -> triggers the rule
>literal string</Component>
```

It's leading me to believe this might have something to do with how the JSX is being parsed, but I can't figure out why. Our eslintrc file includes:

```
...
  "parserOptions": {
    "ecmaFeatures": {
        "jsx": true
    },
    "sourceType": "module"
  },
...
```

also referenced here: https://github.com/yannickcr/eslint-plugin-react/issues/1440

**Comment by ambewas** [comment]
Problem was with the last release of eslint itself. Pinning eslint down to v4.6.1 resolves the issue for now.

I'm closing this and opening an issue with eslint.

**Comment by necolas** [comment]
I'm having a similar problem that I can reproduce with eslint@4.6.1 and v8 of this library
